### PR TITLE
chore(design-system): Component status

### PR DESCRIPTION
**Pull request by frassinier**
**What is the problem this PR is trying to solve?**

Figma, Storybook,  GitHub, and i18n statuses are updated manually twice.

**What is the chosen solution to this problem?**

By completing status per component doc page, we can now aggregate them all on a dedicated page.
image

Demo [here](http://3617.talend.surge.sh/design-system/?path=/docs/component-status--page&globals=theme:light)

**Please check if the PR fulfills these requirements**

- [ ] The PR have used `yarn changeset` to a request a release from the CI if wanted.
- [x] The PR commit message follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md)
- [ ] Tests for the changes have been added (for bug fixes / features) And [non reg](./screenshots.md) done before need review
- [x] Docs have been added / updated (for bug fixes / features)
- [ ] Related design / discussions / pages (not in jira), if any, are all linked or available in the PR

**[ ] This PR introduces a breaking change**

### CODE REVIEW

File: packages/design-system/.storybook/docs/Badge.tsx [hunk 1]
```diff
@@ -0,0 +1,127 @@
+import React from 'react';
+import styled from 'styled-components';
+import tokens from '@talend/design-tokens';
+
+import { Status } from './StatusTable';
+
+const A = styled.a.attrs(({ href }) => ({
```
Comment (Benrajalu) [inline packages/design-system/.storybook/docs/Badge.tsx:7]: What are the benefits of keeping Styled Components as a devDep for the docs components?

Comment (frassinier) [reply packages/design-system/.storybook/docs/Badge.tsx:7]: None, I will fix

Comment (Benrajalu) [reply packages/design-system/.storybook/docs/Badge.tsx:7]: What a man

**Review by jmfrancois — CHANGES_REQUESTED** [review]
legend doesn t fit.
columns name are not the same between v1 and v2
Screenshot 2022-01-18 at 11 53 23
Screenshot 2022-01-18 at 11 53 20

I would like a status on accessibility please :)

**Comment by frassinier** [comment]
> I would like a status on accessibility please :)

@jmfrancois A status on accessibility does not mean anything, actually. 
A11Y is a shared effort between designers, documentation and developers!

File: packages/design-system/.storybook/main.js [hunk 2]
```diff
@@ -27,6 +27,7 @@ module.exports = {
 		buildStoriesJson: true,
 		modernInlineRender: true,
 		previewCsfV3: true,
+		// storyStoreV7: true, // will break all work related to aggregated status in the next major version of Storybook
```
Comment (Benrajalu) [inline packages/design-system/.storybook/main.js:30]: Do we need to keep the commented code + comment? Seems weird to have a warning on top of unused code.

**Review by Benrajalu — APPROVED** [review]

**Comment by Benrajalu** [comment]
> > I would like a status on accessibility please :)
> 
> @jmfrancois A status on accessibility does not mean anything, actually. A11Y is a shared effort between designers, documentation and developers!

Also the React component shouldn't be considered "ok" if not accessible, same for the Figma one.

File: packages/design-system/.storybook/main.js (hunk 2 above)
Comment (frassinier) [reply packages/design-system/.storybook/main.js:30]: I would like to keep that because I need to think about it for the next upgrades if it's ok for you

**Review by jmfrancois — APPROVED** [review]

**Pull request merged**